tutorials/users/catalog/gui-search.md

```diff
@@ -1 +1,115 @@
+    root of the catalog no navigation options are presented here, and you only see the two tabs.
+3. The "Description" section will display the description for the page you are on. In this case it is displaying the _description_ field
+    of the [root catalog](https://daccs.cs.toronto.edu/stac).
+4. The "Additional resources" section includes links to the description and documentation of the STAC API underpinning the catalog.
+    If you are unfamiliar with what this means, then this is not meant for you and you can safely ignore the content.
+5. On the top-right of the screen you always see the "Source" and "Share" tabs. Clicking on the Source tab will show you the version of 
+    [stac specification](https://github.com/radiantearth/stac-spec) that is being used and whether or not the STAC data displayed on that
+    page is valid according to the STAC standards and the [STAC extensions](https://stac-extensions.github.io) used (see image below). Crucially, it provides a quick access to the STAC endpoint 
+    that contains the STAC metadata that is being parsed and displayed by the Browser in any given page.
+    ![Source Tab](images/source-tab.png)
+6. Finally, in the lower part of the document, you will see the "Catalogs" section. This section "lists" all the 
+    child STAC collections, or catalogs, on the node, and provides functionality to sort and search for a catalog by name. In the above image, 
+    there is only only collection, namely the "CMIP6" collection.
+
+```{note}
+Although the root of a STAC catalog has to be a STAC ["catalog"](https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md), 
+there is very little practical difference, from a user's standpoint, whether the children of the root catalog (or their own children, _ad infinitum_) are
+catalogs or STAC ["collection"](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md). In this tutorial, 
+we'll always refer to any child catalog/collection as a "collection", and to the root catalog as "root catalog".
+```
+
+## Inspecting a collection
+Clicking on a collection from the catalog root will take you into a new view that exposes important information about the 
+collection. Here, we are using the "CMIP6" collection as an example.
+
+![Collection View](images/cmip6_collection.png)
+
+1. The "title" section [discussed above](gui_overview) has now changed to the title of the collection.
+2. Below the title, you now have a new "Go to Parent" tab that takes you to the parent catalog, which in this case is the root catalog.
+3. The remainder of the page is laid out in a two-column format: on the left hand side you have information about the collection
+    itself, while on the right hand side you see a list of the items in the collection.
+
+    The left side contains various information about the collection in the following order:
+    ```{attention}
+    This section will be updated after changes to STAC.
```

Comment:
Sounds good. I've also been leaving TODO comments in the tutorials so that I know where to come back to. Something like:

`<!-- TODO: add something important in here later -->`